package.json

```diff
@@ -9,7 +9,9 @@
 		"postinstall": "fumadocs-mdx",
 		"typecheck": "tsc --noEmit",
 		"lint": "biome check",
-		"lint:fix": "biome check --write"
+		"lint:fix": "biome check --write",
+		"generate-dsl-info": "node scripts/generate-dsl-info.mjs",
```

Comment:
write it like `generate:dsl-info` for conventions, so we can do `pnpm run generate:*` in ci